FigureS2.R

```diff
@@ -0,0 +1,79 @@
+source("requirements.R")
+
+glottolog_df <- read_tsv("output/processed_data/glottolog_4.3_languages.tsv", show_col_types = F) %>% 
```

Comment:
Can we put generated text files into ./summary rather than `output` (everything in this repositiory is output).

Reply:
Sure thing!